src/cloudflare/internal/ai-api.ts

```diff
@@ -221,6 +229,68 @@ export class Ai {
     }
   }
 
+  public async toMarkdown(
+    files: { name: string; blob: Blob }[]
+  ): Promise<ConversionResponse[]>;
+  public async toMarkdown(files: {
+    name: string;
+    blob: Blob;
+  }): Promise<ConversionResponse>;
+  public async toMarkdown(
+    files: { name: string; blob: Blob } | { name: string; blob: Blob }[]
+  ): Promise<ConversionResponse | ConversionResponse[]> {
+    const input = Array.isArray(files) ? files : [files];
+
+    const processedFiles = [];
+    for (const file of input) {
+      processedFiles.push({
+        name: file.name,
+        mimeType: file.blob.type,
+        data: btoa(
+          String.fromCharCode(...new Uint8Array(await file.blob.arrayBuffer()))
+        ),
```

Comment:
This feels extremely convoluted. I believe you ought to be able to make use of `internal-node:internal-buffer` here and just use `Buffer.from(await file.blob.arrayBuffer()).toString('base64'))` here.

Reply:
this import as you are suggesting `import { Buffer } from 'internal-node:internal-buffer';` 
throws this error: `src/cloudflare/internal/ai-api.ts(6,24): error TS2307: Cannot find module 'internal-node:internal-buffer' or its corresponding type declarations.`
I didn't find any other `cloudflare/*` file importing anything from `internal-node` so it might be a simple configuration missing

Reply:
The current implementation limits the length of the string to 65k. Using spread operator means that you are bounded by the V8's argument limitation.

Reply:
i would prefer to use the ìnternal-node` version, but I'm unable to import it

Reply:
I wasn't able to import the Buffer class, so i changed the implementation to stop using the spread operator, this should bypass the 65k limit

Reply:
hmm.. yeah it looks like there's a bit more we need to do in the runtime to make importing `node-internal:*` stuff available to `cloudflare-internal:*` ... dang. Ok, will have to fix that then we can update this soon.